draco/programs.py

```diff
@@ -31,3 +31,6 @@ def get_program(file_path: Path) -> Program:
 hard = get_program(asp_path / "hard.lp")
 helpers = get_program(asp_path / "helpers.lp")
 soft = get_program(asp_path / "soft.lp")
+assign_weights = get_program(asp_path / "assign_weights.lp")
```

Comment:
Instead of generating the file, can we instead only do this in memory?

Reply:
removed this line.
`assign_weights` now is an instance variable of `WeightsAssign`